deploy/openshift/template.yaml

```diff
@@ -9,6 +9,77 @@ objects:
     apiVersion: v1
     metadata:
       name: '${SERVICE_ACCOUNT_NAME}'
+  - kind: ServiceAccount
+    apiVersion: v1
+    metadata:
+      name: '${BACKSTAGE_PLUGIN_SA}'
+  - kind: ClusterRole
```

Comment:
I don't think SRE will agree with adding this ClusterRole

Reply:
I have updated the PR as discussed. Could you review it again? Please squash merge the request just to have a clean history. I have multiple commits.